Makefile

```diff
@@ -25,7 +25,7 @@ endif
 ifeq (${STATIC_LIBCPP}, 1)
 	STATIC_LIBCPP=-DSTATIC_LIBCPP=TRUE
 endif
-EXTENSIONS=-DBUILD_PARQUET_EXTENSION=TRUE
+EXTENSIONS=-DBUILD_PARQUET_EXTENSION=TRUE -DBUILD_FTS_EXTENSION=TRUE
```

Comment:
I think it is better to disable it by default to keep the core compilation time small, and manually add the `BUILD_FTS=1` to some of the github actions tests (`.github/workflows/main.yml`) to ensure the extension gets properly tested there (I would set it whereever `BUILD_ICU` is set as well).

Reply:
Done. I left it to build by default in the Python package, though. Thanks for all the feedback!